src/Framework/Assert.php

```diff
@@ -1208,6 +1208,20 @@ public static function assertNotSame($expected, $actual, $message = '')
     }
 
     /**
+     * Refutes that two variables have the same type and value.
+     * Used on objects, it refutes that two variables reference
+     * the same object. Alias for self::assertNotSame().
+     *
+     * @param mixed  $expected
+     * @param mixed  $actual
+     * @param string $message
+     */
+    public static function refuteSame($expected, $actual, $message = '')
+    {
+        return call_user_func_array('self::assertNotSame', func_get_args());
```

Comment:
When I'm aliasing a function/method, I usually do it this way because it is future-compatible with any argument signature changes.  That said, I don't see these arguments changing much in the future, so if you prefer I call it directly, I'm totally fine with that.  Just let me know.